cdc/api/v2/api_helpers.go

```diff
@@ -281,6 +289,56 @@ func (h APIV2HelpersImpl) verifyUpstream(ctx context.Context,
 		return cerror.ErrUpstreamMissMatch.
 			GenWithStackByArgs(cfInfo.UpstreamID, pdClient.GetClusterID(ctx))
 	}
+
+	// check if the upstream cluster has running import tasks.
+	if err := hasRunningImport(ctx, changefeedConfig); err != nil {
+		return cerror.ErrUpstreamHasRunningImport.
+			GenWithStackByArgs(cfInfo.UpstreamID)
+	}
+
+	return nil
+}
+
+// hasRunningImport checks if there is running import tasks on the
+// upstream cluster.
+func hasRunningImport(ctx context.Context, cfg *ChangefeedConfig) error {
+	tlsCfg, err := toolutils.ToTLSConfig(cfg.CAPath, cfg.CertPath, cfg.KeyPath)
+	if err != nil {
+		return err
+	}
+
+	cli, err := etcd.GetEtcdClient(cfg.PDAddrs, tlsCfg)
+	if err != nil {
+		return err
+	}
+
+	resp, err := cli.KV.Get(
+		ctx, RegisterImportTaskPrefix, clientv3.WithPrefix(),
+	)
+	if err != nil {
+		return errors.Annotatef(
+			err, "failed to list import task related entries")
+	}
+
+	for _, kv := range resp.Kvs {
+		leaseResp, err := cli.Lease.TimeToLive(ctx, clientv3.LeaseID(kv.Lease))
+		if err != nil {
+			return errors.Annotatef(
+				err, "failed to get time-to-live of lease: %x", kv.Lease,
+			)
+		}
+		// the lease has expired
+		if leaseResp.TTL <= 0 {
+			continue
+		}
+		return errors.New(
+			"There are lightning/restore tasks running" +
+				"please stop or wait for them to finish. " +
+				"If the lightning/restore task is terminated by system, " +
+				"please wait until the ttl decreases to 0.",
```

Comment:
Please explain what is a `ttl` and how long should be wait in normal cases?

Reply:
add explanation